multiprofit/objects.py

```diff
@@ -1796,20 +1889,17 @@ def getprofiles(self, bandfluxes, engine, cenx, ceny, engineopts=None):
                     raise ValueError("flux ratio not 0 <= {} <= 1".format(fluxratio))
                 flux *= bandfluxes[band]
                 bandfluxes[band] *= (1.0-fluxratio)
-            profile = {}
+            profile = profilebase.copy()
+            profile["axrat"] = axrat
+            profile["ang"] = ang
             for param in self.parameters.values():
-                value = param.getvalue(transformed=False)
-                if param.name == "re":
-                    for modifier in param.modifiers:
-                        if modifier.name == "rscale":
-                            value *= modifier.getvalue(transformed=False)
-                profile[param.name] = value
+                profile[param.name] = param.getvalue(transformed=False)
 
             if not 0 < profile["axrat"] <= 1:
                 if profile["axrat"] > 1:
                     profile["axrat"] = 1
-                elif profile["axrat"] <= 0:
-                    profile["axrat"] = 1e-3
+                elif profile["axrat"] <= __class__.minaxrat:
+                    profile["axrat"] = __class__.minaxrat
                 else:
                     raise ValueError("axrat {} ! >0 and <=1".format(profile["axrat"]))
```

Comment:
In what situation does this raise?  Between `0` and `__class__.minaxrat`?

Reply:
nan will raise, but yes I should change to 0 < x < minaxrat.